ormar/models/newbasemodel.py

```diff
@@ -208,11 +206,6 @@ def __getattr__(self, item: str) -> Any:
             return None
         return super().__getattr__(item)
 
-    def __getattribute__(self, item: str) -> Any:
-        if item == "__dict__":
```

Comment:
yeah that was to check if we can hijack __dict__ calls from typeadapter in fastapi, can be removed but this is one of the ideas how to work around this.